google/ggrc-core
# GGRC-6801 Control instead of mapped object type

# Issue description

Related information popup shows Control instead of mapped object type

# Steps to test the changes

1. Open any audit page
2. Map control and access group to audit
3. Create assessment with Control type
4. Open Assessment info panel> Other attributes tab
5. Click Map icon for Related Information and map Access group snapshot to assessment
6. Click on Access group snapshot to open popup screenshot-1.png
Actual Result: Related information popup shows Control instead of Access group
Expected Result: Related information popup should show mapped object type

7. Look at the DESCRIPTION
Actual Result: Layout of the description is incorrect screenshot-2.png
Expected Result: Improve Layout of the description. Refer to UX/BA team.

# Solution description

There're 2 separate bugs here: template bug and minor CSS problem.

The template bug fix is simple but strange: I just changed title text from assessmentType to title of the instance class of this modal. I have no idea why there was assessment type if we need the title of the instance class.

CSS is fixed simply by adding margin-top to the wrapper of the modal content

# Sanity checklist


# PR Review checklist
